.github/workflows/ci.yml

```diff
@@ -51,10 +48,52 @@ jobs:
         run: |
           cd ccm
           sudo ./setup.py install
-      - name: run tests and analyze
+      - name: run tests
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
           CCM_CLUSTER_START_TIMEOUT_OVERRIDE: 600
-        run: mvn -B verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar --file pom.xml
+        run: >
+          mvn -B verify --file pom.xml -P it-${{ matrix.backend }}
+
+  sonar:
```

Comment:
If you want you can drop this entire block since sonar is being removed

Reply:
ok